README.md

```diff
@@ -14,6 +14,22 @@ npm run build
 
 ## Deployment
 
+### Automated Deployment
+
+We now have a series of scripts that will help with automating the release process. To use these scripts, do the following.
+
+1. Run `npm install` in the project root to make sure all dependencies are installed at the project level and in the `./scripts` folder.
+1. Run `npm run bump-version` in the project root. This will increment the project version and create a git commit.
+1. Run `node scripts/commands/readme.js` in the project root. This will prompt you to select the commits that are part of this release. The commit log messages will be automatically added to the [changelog](readme.txt) and committed.
```

Comment:
Should we checkout  a working branch first? Otherwise we're going to end up (auto)committing changes to the `trunk/master` branch.

Reply:
The bump-version command automatically creates a new branch.